xwiki-commons-core/xwiki-commons-environment/xwiki-commons-environment-servlet/src/main/java/org/xwiki/environment/internal/ServletEnvironment.java

```diff
@@ -54,6 +62,41 @@ public class ServletEnvironment extends AbstractEnvironment
      */
     private javax.servlet.ServletContext javaxServletContext;
 
+    @Inject
+    private ComponentManager componentManager;
```

Comment:
I think it's ok but just to be sure: we agree it wouldn't make sense to use the contextual CM here? (`@Named("context")`)?

Reply:
I don't think it would make much sense to use the context since that componentManager is used way before any concept of context event exist.

Reply:
All good then.